dotcom-rendering/src/components/AdSlot.tsx

```diff
@@ -137,23 +139,121 @@ export const labelStyles = css`
 	}
 `;
 
-const adContainerCollapseStyles = css`
-	& .ad-slot.ad-slot--collapse {
+const adSlotCollapseStyles = css`
+	&.ad-slot.ad-slot--collapse {
```

Comment:
We apply `ad-slot-collapse` to the slots (not containers) [here](https://github.com/guardian/commercial/blob/main/src/lib/dfp/render-advert.ts#L100). So I think if we are going to only have one collapse style we should apply it to slots and not containers.

We could rethink how we do collapsing slots to apply it to the container level but that is getting outside the scope of this PR.

Reply:
Yeah you're right, I'll revert this change